aspnetcore/security/authentication/identity.md

```diff
@@ -368,7 +368,7 @@ Post is specified in the *Pages/Shared/_LoginPartial.cshtml*:
 
 ## Test Identity
 
-The default web project templates allow anonymous access to the home pages. To test Identity, add [`[Authorize]`](/dotnet/api/microsoft.aspnetcore.authorization.authorizeattribute) to the Privacy page.
+The default web project templates allow anonymous access to the home pages. To test Identity, add [[Authorize]](/dotnet/api/microsoft.aspnetcore.authorization.authorizeattribute) to the Privacy page.
```

Comment:
I think these look better code fenced. We're more flexible on that that the style guide.

Reply:
Perhaps one to two dozen of these that have brackets in link text were done this way (no styling) ... and these few that I updated here had styling. There was at least an inconsistency present within the repo.